IPv6 sending without scope is broken in tests

Regressed in #343

```
DEBUG    zeroconf:_core.py:667 Sending to (None, 5353) (34 bytes #1) <DNSOutgoing:{multicast=True, flags=0, questions=[ptr[question,QM,in,_http._tcp.local.]], answers=[], authorities=[], additionals=[]}> as b'\x00\x00\x00\x00\x00\x01\x00\x00\x00\x00\x00\x00\x05_http\x04_tcp\x05local\x00\x00\x0c\x00\x01'...
DEBUG    zeroconf:_logger.py:47 Error sending through socket 16
Traceback (most recent call last):
  File "/Users/runner/work/python-zeroconf/python-zeroconf/zeroconf/_core.py", line 680, in async_send
    transport.sendto(packet, (real_addr, port or _MDNS_PORT, *v6_flow_scope))
TypeError: 'NoneType' object is not iterable
```

--- a/tests/services/test_browser.py
+++ b/tests/services/test_browser.py
@@ -447,7 +447,7 @@ def test_backoff(suppresses_mock):
         """Current system time in milliseconds"""
         return start_time + time_offset * 1000
 
-    def send(out, addr=const._MDNS_ADDR, port=const._MDNS_PORT, v6_flow_scope=None):
+    def send(out, addr=const._MDNS_ADDR, port=const._MDNS_PORT, v6_flow_scope=()):
         """Sends an outgoing packet."""
         got_query.set()
         old_send(out, addr=addr, port=port, v6_flow_scope=v6_flow_scope)
@@ -679,7 +679,7 @@ def test_integration():
     expected_ttl = const._DNS_HOST_TTL
     nbr_answers = 0
 
-    def send(out, addr=const._MDNS_ADDR, port=const._MDNS_PORT, v6_flow_scope=None):
+    def send(out, addr=const._MDNS_ADDR, port=const._MDNS_PORT, v6_flow_scope=()):
         """Sends an outgoing packet."""
         pout = r.DNSIncoming(out.packets()[0])
         nonlocal nbr_answers

--- a/tests/services/test_info.py
+++ b/tests/services/test_info.py
@@ -210,7 +210,7 @@ class TestServiceInfo(unittest.TestCase):
 
         last_sent = None  # type: Optional[r.DNSOutgoing]
 
-        def send(out, addr=const._MDNS_ADDR, port=const._MDNS_PORT, v6_flow_scope=None):
+        def send(out, addr=const._MDNS_ADDR, port=const._MDNS_PORT, v6_flow_scope=()):
             """Sends an outgoing packet."""
             nonlocal last_sent
 
@@ -355,7 +355,7 @@ class TestServiceInfo(unittest.TestCase):
 
         last_sent = None  # type: Optional[r.DNSOutgoing]
 
-        def send(out, addr=const._MDNS_ADDR, port=const._MDNS_PORT, v6_flow_scope=None):
+        def send(out, addr=const._MDNS_ADDR, port=const._MDNS_PORT, v6_flow_scope=()):
             """Sends an outgoing packet."""
             nonlocal last_sent
 

